data/yaml/venues/ml4al.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: ML4AL
+name: 1st Workshop on Machine Learning for Ancient Languages
```

Comment:
~~1st~~ Workshop on Machine Learning for Ancient Languages

Reply:
Done.